src/plugins/intel_gna/src/transformations/matops_to_dwsc.cpp

```diff
@@ -0,0 +1,261 @@
+                return false;
+            auto new_transpose = std::make_shared<ngraph::op::Transpose>(
+                math_node->input_value(0),
+                ngraph::op::Constant::create(element::Type_t::i64, Shape{2}, {1, 0}));
+            input_4d = std::make_shared<ngraph::opset1::Reshape>(
+                new_transpose->output(0),
+                ngraph::op::Constant::create(ngraph::element::i64, Shape{4}, {N, C, H, W})->output(0),
+                false);
+        } else
+            input_4d = std::make_shared<ngraph::opset1::Reshape>(
+                math_node->input_value(0),
+                ngraph::op::Constant::create(ngraph::element::i64, Shape{4}, {N, C, H, W})->output(0),
+                false);
+        new_dwsc = std::make_shared<ngraph::opset1::GroupConvolution>(input_4d->output(0),
+                                                                      dwsc_weights_const->output(0),
+                                                                      strides,
+                                                                      pads_begin,
+                                                                      pads_end,
+                                                                      dilations,
+                                                                      auto_pad);
+    } else {
+        new_dwsc = std::make_shared<ngraph::opset1::GroupConvolution>(math_node->input_value(0),
+                                                                      dwsc_weights_const->output(0),
+                                                                      strides,
+                                                                      pads_begin,
+                                                                      pads_end,
+                                                                      dilations,
+                                                                      auto_pad);
+    }
+
+    new_dwsc->set_friendly_name("replace_math_operation");
+
+    std::shared_ptr<ov::Node> skip_node;
+    if (nullptr == std::dynamic_pointer_cast<ngraph::opset1::Multiply>(math_node)) {
+        // creating a bias node
+        auto bias_const =
+            std::dynamic_pointer_cast<ngraph::opset1::Constant>(math_node->input_value(1).get_node_shared_ptr());
+        const float* bias_ptr = bias_const->get_data_ptr<float>();
+        std::vector<float> new_bias(N_params * C_params * H_params * W_params, 0.0f);
+        float* new_bias_ptr = new_bias.data();
+
+        if (nullptr == std::dynamic_pointer_cast<ngraph::opset1::Subtract>(math_node)) {
+            for (size_t i = 0; i < N_params * C_params * H_params * W_params; i++)
+                *(new_bias_ptr + i) = *(bias_ptr + i);
+        } else {
+            for (size_t i = 0; i < N_params * C_params * H_params * W_params; i++)
+                *(new_bias_ptr + i) = -1 * (*(bias_ptr + i));
```

Comment:
As above (e.g. std::transform could be used)

Reply:
Done